bin/app.ts

```diff
@@ -8,7 +8,11 @@ class AwsCostBotStack extends cdk.Stack {
       description: "https://github.com/bwobbones/aws-cost-bot"
     });
 
-    new AwsCostBot(this, "AwsCostBot");
+    new AwsCostBot(this, "AwsCostBot", {
+      configFile:
+        process.env.CONFIG_FILE ||
+        "arn:aws:s3:::generalresourceful/rsfl-environments.json"
```

Comment:
@StevenGBrown lets keep this repo public friendly, probably just throw a descriptive error if there is no `CONFIG_FILE`

Reply:
Sure thing, 2d7a6d8